pkgs/applications/version-management/git-and-tools/git-remote-hg/default.nix

```diff
@@ -1,34 +1,37 @@
-{ stdenv, fetchgit, mercurial, makeWrapper,
+{ stdenv, fetchFromGitHub, mercurial, python2Packages,
   asciidoc, xmlto, docbook_xsl, docbook_xml_dtd_45, libxslt, libxml2
 }:
 
-stdenv.mkDerivation rec {
-  rev = "e716a9e1a9e460a45663694ba4e9e8894a8452b2";
-  version = "0.2-${rev}";
-  name = "git-remote-hg-${version}";
+python2Packages.buildPythonApplication rec {
+  version = "1.0.0";
+  pname = "git-remote-hg";
 
-  src = fetchgit {
-    inherit rev;
-    url = "git://github.com/fingolfin/git-remote-hg.git";
-    sha256 = "0cmlfdxfabrs3x10mfjfap8wz67s8xk2pjn2wlcj9k2v84gji60m";
+  # not using fetchPypi - the PyPI tarball does not include the manpage
+  src = fetchFromGitHub {
+    owner = "mnauw";
+    repo = "git-remote-hg";
+    rev = "v${version}";
+    sha256 = "0anl054zdi5rg5m4bm1n763kbdjkpdws3c89c8w8m5gq1ifsbd4d";
   };
 
-  buildInputs = [ mercurial.python mercurial makeWrapper
-    asciidoc xmlto docbook_xsl docbook_xml_dtd_45 libxslt libxml2
-  ];
+  nativeBuildInputs = [ asciidoc xmlto docbook_xsl docbook_xml_dtd_45 libxslt libxml2 ];
+  propagatedBuildInputs = [ mercurial ];
 
   doCheck = false;
```

Comment:
Has tests https://github.com/mnauw/git-remote-hg/blob/master/Makefile#L10

Maybe try to enable them.

Reply:
Looks like the tests are going to be problematic. They mostly pass but one or two of them fail. What's worse, the failures are different with different versions of Git and Mercurial. I'll have to look into it.